docs/howtos/expanding-federation.md

```diff
@@ -0,0 +1,44 @@
+# How to add a new node to the federation
+
+Adding a new node to the checkpointing federation requires every Morpho node to make changes to their configuration and restarting the Morpho network. In addition, every proof-of-work client in the network will also have to adjust their configuration. This document explains the theoretical process of what needs to be done for this to work.
```

Comment:
In addition -> Also
will also have to -> will have to
restarting the Morpho network -> the Morpho network must be restarted
Delete 'theoretical' - this description gives practical steps as well as the explanation.